<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminlogin</class>
 <widget class="QWidget" name="adminlogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1,
                            stop:0 #e3f2fd, stop:1 #bbdefb);
</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>250</y>
     <width>501</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SizeConstraint::SetMaximumSize</enum>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>10</number>
    </property>
    <item row="5" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="pushButton_adminverificationback">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
background-color: #7F8C8D;  /* Medium gray */
color: white;

font-weight: bold;
    border-radius: 5px;
    padding: 5px 10px;
}

QPushButton:hover {
background-color: #626567;  /* Darker gray */

}
</string>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
        <property name="icon">
         <iconset theme="QIcon::ThemeIcon::GoPrevious"/>
        </property>
        <property name="iconSize">
         <size>
          <width>13</width>
          <height>12</height>
         </size>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_admin_ui_login">
        <property name="styleSheet">
         <string notr="true">QPushButton {
 background-color: #27AE60;  /* Emerald green */
    background-color: #2ECC71;  /* Light green */
;
font-weight: bold;
    color: white;
    border-radius: 5px;
    padding: 5px 10px;
}

QPushButton:hover {
background-color: #1E8449;  /* Darker shade of emerald green */

}
</string>
        </property>
        <property name="text">
         <string>Login</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2_admin_password">
        <property name="styleSheet">
         <string notr="true">
QLabel {
background-color: #3498DB;  /* Medium gray */
color: white;

    border-radius: 5px;
    padding: 5px 10px;
font-size: 10px;
    font-weight: bold;
}
</string>
        </property>
        <property name="text">
         <string>Passwrord:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_2_adminpassword">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    background-color: #ECF0F1;
    color: #2C3E50;
    border: 1px solid #BDC3C7;
    border-radius: 3px;
    padding: 2px 4px;
    font-size: 12px;
    min-height: 20px; /* reduce height */
}

QLineEdit:focus {
    border: 1px solid #3498DB;
    background-color: #FFFFFF;
}

QLineEdit::placeholder {
    color: #95A5A6;
    font-style: italic;
}
</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::EchoMode::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_admin_name">
        <property name="styleSheet">
         <string notr="true">
QLabel {
background-color: #3498DB;  /* Medium gray */
color: white;

    border-radius: 5px;
    padding: 5px 10px;
font-size: 10px;
    font-weight: bold;
}
</string>
        </property>
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_adminname">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    background-color: #ECF0F1;
    color: #2C3E50;
    border: 1px solid #BDC3C7;
    border-radius: 3px;
    padding: 2px 4px;
    font-size: 12px;
    min-height: 20px; /* reduce height */
}

QLineEdit:focus {
    border: 1px solid #3498DB;
    background-color: #FFFFFF;
}

QLineEdit::placeholder {
    color: #95A5A6;
    font-style: italic;
}
</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">

QLabel {
background-color:#5D6D7E;  /* Medium gray */
color: white;

    border-radius: 5px;
    padding: 5px 10px;
font-size: 12px;
    font-weight: bold;
}
</string>
      </property>
      <property name="text">
       <string>Admin Verification:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QDateTimeEdit" name="dateTimeEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>label_admin_name</sender>
   <signal>linkHovered(QString)</signal>
   <receiver>lineEdit_adminname</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>118</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>label_2_admin_password</sender>
   <signal>linkHovered(QString)</signal>
   <receiver>lineEdit_2_adminpassword</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
